coast/data/tidegauge.py

```diff
@@ -93,8 +93,31 @@ def __init__(self, dataset=None, config: Union[Path, str] = None, new_time_coord
     ############ tide gauge methods ###########################################
     def read_gesla_v3(self, fn_gesla, date_start=None, date_end=None):
         """
-        For reading from a GESLA2 (Format version 3.0) file(s) into an
-        xarray dataset. Formatting according to Woodworth et al. (2017).
+        Depreciated method.
+        Call generalised method.
+        Returns eiter a tidegauge object or a list of tidegauge objects
+        """
+        # See if its a file list input, or a glob
+        if type(fn_gesla) is not list:
```

Comment:
Python tip: A better practice for testing the type of an object in python is the use the isinstance method. 
E.g.:
```python
if not isinstance(fn_gesla, list):
```